runtime/metricsview/query.go

```diff
@@ -124,6 +124,7 @@ type TimeSpine struct {
 	Start time.Time `mapstructure:"start"`
 	End   time.Time `mapstructure:"end"`
 	Grain TimeGrain `mapstructure:"grain"`
+	Alias string    `mapstructure:"alias"`
```

Comment:
What does this do? Ideally it should apply only to the time dimension if it's requested, and in that case, the time dimension already has an alias provided in `Dimension`